packages/nimble-components/src/anchor-menu-item/specs/README.md

```diff
@@ -0,0 +1,145 @@
+
+GitHub menu with links:
+
+![GitHub](github.png)
+
+---
+
+## Design
+
+### API
+
+_The key elements of the component's public API surface:_
+
+-   _Component Name_: `nimble-anchor-menu-item`
+-   _Props/Attrs_:
+    -   `download` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `href` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes).
+    -   `hreflang` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `ping` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `referrerpolicy` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `rel` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `target` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `type` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `disabled` - disables the item (this matches the API of `nimble-menu-item`). Setting it causes `aria-disabled` to be set as well.
+-   _Methods_
+-   _Events_
+-   _CSS Classes and CSS Custom Properties that affect the component_
+-   _Slots_:
+    -   We will provide a `start` slot, just like is available on the `nimble-menu-item`. That component also has an `end` slot, but we do not document it, so we will not define an `end` slot on the anchor menu item.
+
+### Anatomy
+
+```html
+<template role="menuitem" aria-disabled="${x => x.disabled}">
+    <a
+        download="${x => x.download}"
+        href=${x => (x.disabled ? null : x.href)}
+        hreflang="${x => x.hreflang}"
+        ping="${x => x.ping}"
+        referrerpolicy="${x => x.referrerpolicy}"
+        rel="${x => x.rel}"
+        target="${x => x.target}"
+        type="${x => x.type}"
+    >
+        ${startSlotTemplate(options)}
+        <slot></slot>
+    </a>
+</template>
+```
+
+### Angular integration
+
+We will follow the same patterns as we've used for other anchor-based components, like the `nimble-anchor-button`. We will have a directive for the component, but no CVA. We will also have two directives, `NimbleAnchorMenuItemRouterLinkWithHref` and `NimbleAnchorMenuItemRouterLink`. The former will support use of a `nimbleRouterLink` attribute, and the latter will prevent the use of `routerLink`. There will be no support for `routerLinkActive`, which fortunately doesn't seem like a very useful thing for an anchor menu item.
+
+### Blazor integration
+
+We will create a Blazor wrapper for the anchor menu item. There are no special routing considerations/mechanisms for Blazor.
+
+### Visual Appearance
+
+The component will look just like a standard menu item. See the design doc linked at the top of this file for details.
+
+---
+
+## Implementation
```

Comment:
Anything we need to think about / document about the expectations that the nimble-menu has about its items? It doesn't encode any direct dependencies on nimble-menu-item, does it?

Reply:
Not that I can find.